server.ts

```diff
@@ -53,10 +53,22 @@ app.get(
   '/api/config',
   jsonHandler(async () => {
     const hubDiscoveryUrl = new URL(`http://${hubHost}/emulators`);
-    const emulatorsRes = await fetch(hubDiscoveryUrl.toString());
-    const emulators = (await emulatorsRes.json()) as any;
+    const experimentDiscoveryUrl = new URL(
+      `http://${hubHost}/enabled-experiments`
```

Comment:
Does this endpoint still exist on the CLI side?

Reply:
Updated to remove this call and use env var instead